data/hai/hai outfits.txt

```diff
@@ -11,6 +11,37 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Phantom Pallet"
+	category "Systems"
+	cost 12000
+	thumbnail "outfit/phantom pallet"
+	"mass" 0.2
+	"unplunderable" 1
+	"energy consumption" 0.016667
+	"heat generation" 0.03334
```

Comment:
```suggestion
	"energy consumption" 0.015
	"heat generation" 0.03
```
We generally try to avoid having these kinds of approximated recurring decimals in the files.